core/src/main/java/org/opensearch/sql/expression/aggregation/StdDevAggregator.java

```diff
@@ -1,16 +1,9 @@
 /*
- *   Licensed under the Apache License, Version 2.0 (the "License").
- *   You may not use this file except in compliance with the License.
- *   A copy of the License is located at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *   or in the "license" file accompanying this file. This file is distributed
- *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- *   express or implied. See the License for the specific language governing
- *   permissions and limitations under the License.
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
We need to confirm that this file wasn't copied from another project. In that case we need to preserve the license header (or keep both).

Reply:
Looks like it was part of 254510e5 `254510e554156ef83365e0ea68935d4d75145330`

By David Cui 2021-06-25 5:47 p.m.

Reply:
I found only one matching file on GH (name + license), but code is completely different.
https://github.com/apache/druid/blob/master/extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/VarianceAggregator.java
https://github.com/apache/druid/blob/master/extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/StandardDeviationPostAggregator.java